README.md

```diff
@@ -379,17 +411,101 @@ Documentation Forthcoming
 
 ### Modules
 
-#### `loadModuleData`
+#### Loading Modules
 
-Documentation Forthcoming
+[Holocron Modules](#modules) are loaded and composed on 1) the Server when serving the HTML document and 2) in the Browser [Single Page App](https://developer.mozilla.org/en-US/docs/Glossary/SPA) mode via AJAX. We may load Holocron Modules when 1) a path is matched to the URL, or 2) dispatching a Redux action using an HOC and rendering with a React Component.
+
+**Contents**
+* React Router Component
+  * [`ModuleRoute`](#moduleroute)
+* Load and Render
+  * [`RenderModule`](#rendermodule)
+  * [`holocronModule`](#holocronmodule)
+  * [`composeModules`](#composemodules)
+
+##### React Router Component
+
+> 👍 Most commonly used method to load Holocron Modules
+
+A parent Module may add the `ModuleRoute` routing component to the [`childRoutes` Module Lifecycle Hook](#routing) to load a child Module dynamically on the server or browser when matching a route path. Once the Module is loaded, it is injected as a JSX element into the `children` prop of the parent Module.
+
+###### `ModuleRoute`
+
+<!-- ONE-MARKDOWN:START url="https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md" selector="### `ModuleRoute`" -->
+Please see [`ModuleRoute`](https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route#-usage) in the Holocron Module Route API.
+<!-- ONE-MARKDOWN:END -->
+
```

Comment:
These sections are sparse as we decided to link to API methods that belong to other Libraries. I included a sample comment of how I envision the One Markdown module to work to "stitch" in API method definitions from other markdown files. If something goes wrong, it displays the contents of the "ONE-MARKDOWN:START" and "ONE-MARKDOWN:END" comment directives.